libraries/eosiolib/core/eosio/name.hpp

```diff
@@ -313,11 +313,7 @@ namespace eosio {
    } /// namespace detail
 } /// namespace eosio
 
-/// @cond IMPLEMENTATIONS
```

Comment:
@larryk85 I have to delete this annotation cause it is preventing doxygen from parsing the annotation for `_n()` operator and thus the operator is not documented in the reference documentation anymore

Reply:
Sure, that sounds fine.